pkg/kubectl/doc.go

```diff
@@ -14,7 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package kubectl is a set of libraries that are used by the kubectl command line tool.
-// They are separated out into a library to support unit testing.  Most functionality should
-// be included in this package, and the main kubectl should really just be an entry point.
+// Package kubectl provides the functions used by the kubectl command line tool
+// under /cmd/. The functions are kept in this package to better support unit
```

Comment:
Do you actually mean `/cmd/` or `/cmd/kubectl`? I also somehow find that first slash off: `cmd/` ?